lerna.json

```diff
@@ -1,6 +1,11 @@
 {
     "lerna": "2.5.1",
     "packages": ["packages/*"],
+    "commands": {
+        "publish": {
+            "allowBranch": "development"
+        }
+    },
```

Comment:
Why was this added?

Reply:
This prevents us from accidentally publishing from another branch. Just a safety check